Frontend/src/Components/CreateListing/CreateListing.jsx

```diff
@@ -80,22 +82,30 @@ const CreateListing = () => {
             handlePhotoDelete={handlePhotoDelete}
           />
           <DescriptionSection
-            title={title}
-            setTitle={setTitle}
-            description={description}
-            setDescription={setDescription}
+            title={formInput.title}
+            setTitle={(value) => setFormInput({ ...formInput, title: value })}
```

Comment:
nit: instead of passing in several set functions, you can just pass in setFormInput and handle setTitle and setDescription within the DescriptionSection component

Reply:
the same thing can be said about passing in title and description separately -- you can just pass in formInput in this scenario